test/tests/functional/pbs_release_limited_res_suspend.py

```diff
@@ -756,3 +756,28 @@ def test_resuming_excljob_with_no_res_released(self):
         # resume job
         self.server.deljob(jid2, wait=True)
         self.server.expect(JOB, {ATTR_state: 'R'}, id=jid1)
+
+    def test_normal_user_unable_to_see_res_released(self):
+        """
+        Check if normal user (non-operator, non-manager) has priveleges to see
+        resources_released and resource_released_list attribute in job status
+        """
+        # Set mem in restrict_res_to_release_on_suspend server attribute
+        a = {ATTR_restrict_res_to_release_on_suspend: 'mem'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        j1 = Job(TEST_USER)
+        j1.set_attributes({ATTR_l + '.select': '1:ncpus=4:mem=512mb'})
+        jid1 = self.server.submit(j1)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid1)
+        # suspend job
+        self.server.sigjob(jobid=jid1, signal="suspend")
+        self.server.expect(JOB, {ATTR_state: 'S'}, id=jid1)
+
+        # stat the job as a normal user
+        attrs = self.server.status(JOB, id=jid1, runas=TEST_USER)
+        self.assertFalse("resources_released" in attrs,
+                         "normal user can not see read resources_released")
+
```

Comment:
Please get rid of "read" to make this failure message similar to that below for resource_released_list.mem.